test/e2e/framework/volume_util.go

```diff
@@ -329,9 +328,27 @@ func StartVolumeServer(client clientset.Interface, config VolumeTestConfig) *v1.
 			ExpectNoError(err, "Cannot locate the server pod %q: %v", serverPodName, err)
 		}
 	}
+	if config.ServerReadyMessage != "" {
+		err := waitForPodLog(client, pod, serverPodName, config.ServerReadyMessage)
+		ExpectNoError(err, "Failed to find %q in pod logs: %s", config.ServerReadyMessage, err)
+	}
 	return pod
 }
 
+func waitForPodLog(client clientset.Interface, pod *v1.Pod, containerName string, message string) error {
```

Comment:
should this func be moved to _framework/util.go_ since it's more generic that just for volumes?